.github/workflows/main.yaml

```diff
@@ -15,4 +15,4 @@ jobs:
       uses: cachix/install-nix-action@v12
 
     - name: Run tests
-      run: nix-shell --run ./run-tests.py
+      run: nix run -c ./run-tests.py
```

Comment:
Just to save myself confusion in the future, Nix 2.4 will likely use `nix shell` command for this feature (this is how it is named at current nixUnstable).